PGPAuth/src/main/java/org/lf_net/pgpunlocker/Logic.java

```diff
@@ -71,12 +71,12 @@ public void close() {
         }
     }
 
-    public void doActionOnServerWithFeedback(String action, String server, String keyAPG) {
-        String ret = doActionOnServer(action, server, keyAPG);
+    public void doActionOnServerWithFeedback(String action, String server, String keyAPG, String openPGPKey) {
```

Comment:
please rename to `openPGPKeyID` everywhere

This way its is clearer what that variable is ^^